specifications/M1/dsp.profile.md

```diff
@@ -0,0 +1,49 @@
+# 1. Introduction
+
+This document defines a profile of
+the [Dataspace Protocol Specifications (DSP)](https://github.com/International-Data-Spaces-Association/ids-specification).
+
+# Catalog Metadata for the __Well-Known__ Endpoint
+
+Catalogs that support the [Credential Issuance Protocol (CIP)](credential.issuance.protocol.md) MUST return a JSON
+object from the `/.well-known/dspace-trust` endpoint with the following properties:
+
+- `@context`: REQUIRED. Specifies a valid [Json-Ld context](https://www.w3.org/TR/json-ld11/#the-context).
+- `credentialsSupported`: REQUIRED. Contains a Json structure corresponding to the schema
+  specified [below](#the-credentialssupported-object).
+
+A non-normative example is:
+
+```json
+{
+  "@context": [
+    ...
+  ],
+  "credentialsSupported": [
+    {
+      "types": [
+        "VerifiableCredential",
+        "CompanyCredential"
+      ],
+      "bindingMethod": "did:web",
```

Comment:
minor nit: we should use plural, and reflect that in the JSON example, or use singular to maintain consistency. If plural, can you elaborate, what the semantics are then, "any-of" vs "all-of"? 

```suggestion
      "bindingMethods": [ "did:web" ],
```

Reply:
I don't think we should support more than one binding method per VC entry since this can be done with multiple entries and probably a very, very remote scenario. I would keep it as-is.